src/Nethermind/Nethermind.Synchronization/ParallelSync/SyncDispatcher.cs

```diff
@@ -160,6 +128,49 @@ public async Task Start(CancellationToken cancellationToken)
             }
         }
 
+        private async Task DoDispatch(CancellationToken cancellationToken, PeerInfo? allocatedPeer, T request,
+            SyncPeerAllocation allocation)
+        {
+            try
+            {
+                await Dispatch(allocatedPeer, request, cancellationToken);
+            }
+            catch (ConcurrencyLimitReachedException)
+            {
+                if (Logger.IsDebug) Logger.Debug($"{request} - concurrency limit reached. Peer: {allocatedPeer}");
```

Comment:
We just ignore it, will it be re-dispatched later?

Reply:
We already ignore it currently... so yes?